.github/workflows/nightly.yml

```diff
@@ -79,30 +88,78 @@ jobs:
             pytorch_channel: pytorch-nightly
           - pytorch_version: 1.6
             pytorch_channel: pytorch-nightly
+          - pytorch_version: 1.3
+            pytorch_channel: pytorch
+            python_version: 3.8
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
+      # https://github.com/docker/setup-buildx-action
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
       - name: Publish CUDA to Docker Hub
         # publish master
-        uses: docker/build-push-action@v1.1.0
+        uses: docker/build-push-action@v2
+        with:
+          build-args: PYTHON_VERSION=${{ matrix.python_version }},PYTORCH_VERSION=${{ matrix.pytorch_version }},PYTORCH_CHANNEL=${{ matrix.pytorch_channel }}
+          cache-from: pytorchlightning/pytorch_lightning:base-cuda-cache-py${{ matrix.python_version }}-torch${{ matrix.pytorch_version }}
+          cache-to: pytorchlightning/pytorch_lightning:base-cuda-cache-py${{ matrix.python_version }}-torch${{ matrix.pytorch_version }}
+          file: dockers/base-cuda/Dockerfile
+          push: true
+          tags: pytorchlightning/pytorch_lightning:base-cuda-py${{ matrix.python_version }}-torch${{ matrix.pytorch_version }}
+        timeout-minutes: 55
+
+  docker-Conda:
```

Comment:
lets keep this in single run, sowe do not think twire if cuda and conda is paired

Reply:
Yes, done!

Reply:
well, I still see it here :]